src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -434,6 +434,20 @@ protected function fixManagersAutoMappings(array $managerConfigs, array $bundles
      */
     abstract protected function getMappingResourceExtension();
 
+    /**
+     * The class name used by the various mapping drivers.
+     *
+     * @param string $driverType
+     *
+     * @return string
+     */
+    protected function getMetadataDriverClass($driverType)
```

Comment:
I think you can use type declaration and return type

Reply:
Really? Symfony targets PHP 5, doesn’t it?

Reply:
I just saw it doesn’t, silly me. I’ll update it.